pages/verse/verse.md

```diff
@@ -0,0 +1,61 @@
+---
+# Name of the project. Also the URL i.e. duneanalytics.com/projects/dai.
+name: “VERSE” 
+
+# The name of the logo file. Should have the same name as the markdown file.
+logo: "verse-black.svg"
+
+# The main color of the project. Remember to quote this value.
+color: "#FFFFFF”
+
+# List of relevant schemas you can query in dune.
+schemas: 
+  - “verse”
+
+# List of relevant links to the project.
+links:
+  - name: "Website"
+    link: "https://www.bitcoin.com/"
+  - name: “Verse DEX”
+    link: "https://verse.bitcoin.com”
+  - name: “Whitepaper”
+    link: "https://www.getverse.com/verse-whitepaper.pdf"
+  - name: “Github”
+    link: "https://github.com/bitcoin-portal/bitcoincom-solidity-swap"
+
+# List of applicable tags in Dune. These will link to e.g...
+tags:
+  - “verse”
+  - “ethereum"
+  - “bcom”
+  - “bch”
+  - "smartbch"
+  - “bitcoin”
+  - "DeFi"
+  - “DEX” 
+  
+# List of related projects. These projects must exist in this repo and the name...
+related: 
+  - “verse” 
+
+# List of relevant dashboard slugs prefixed with username
+dashboards:
+  - "alun/verse"
+  - "jtk/verse-staking"
+  - "Henrystats/verse-farmers-analytics"
+
+# List of relevant query-ids.
+queries:
+  - 1942119
+
+# List of visualizations to embed
+embeds:
+  - name: “Verse Holders over time”
+    link: "https://dune.com/queries/1942119/3204019" 
```

Comment:
this needs to be an embed, eg: 
`https://dune.com/embeds/31193/62877/a7284a40-96e1-484b-a99b-cc00c25f16b5`